README.md

```diff
@@ -30,16 +30,16 @@ Run the app in development mode using production APIs (you don’t need to start
 yarn dev
 ```
 
-While the web server is running, you can open [http://localhost:5000](http://localhost:5000) in your browser to view the app.
+While the web server is running, you can open [http://localhost:3000](http://localhost:3000) in your browser to view the app.
 To stop the server, press `CTRL+C` in the terminal.
 
 ---
 
 If you want to use local API endpoints instead of the default remote ones, create a new file called `.env.local` with the following contents:
 
 ```ini
-NEXT_PUBLIC_API_JS=http://localhost:2000/api/v1
-NEXT_PUBLIC_API_PY=http://localhost:3000/stat/v1
+NEXT_PUBLIC_API_JS=http://localhost:4000/api/v1
+NEXT_PUBLIC_API_PY=http://localhost:4010/stat/v1
```

Comment:
@saifulkhan I picked pretty much random ports here, so happy to follow your advice if you want do change them. The reason why I used 4000+ is because I saw this range of ports used by GraphQL servers in the past ([example](https://graphql.org/graphql-js/running-an-express-graphql-server/)). However, I’m not sure if this is a standard convention for APIs or a local choice.

If we keep port 5000 free, we can use [`serve`](https://www.npmjs.com/package/serve) without having to configure it. A flow can be something like:

```sh
npm install --global serve

cd /path/to/ui
yarn build
serve out
## → local prod build on http://localhost:5000
```

Reply:
@kachkaev, sounds good to me, thank you.